src/main/java/com/direwolf20/buildinggadgets/common/tools/ToolRenders.java

```diff
@@ -121,7 +121,13 @@ public static void renderBuilderOverlay(RenderWorldLastEvent evt, EntityPlayer p
                     return;
                 }
                 if (coordinates.size() == 0 && lookingAt != null) { //Build a list of coordinates based on the tool mode and range
-                    coordinates = BuildingModes.getBuildOrders(world, player, lookingAt.getBlockPos(), lookingAt.sideHit, heldItem);
+                    BlockPos block;
+                    if (flIntState == LiquidInteractions.GadgetBuilding.REPLACE_TOP) {
```

Comment:
Make this a defaulted Method on the enum... It's the buisness of the enum to decide, whether you need to build a bit displaced...